hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -118,7 +118,7 @@ public final class Hedera implements SwirldMain {
     private static final int VERSION_NOT_IN_SAVED_STATE_EXIT_CODE = 11;
     private static final int CRITICAL_FAILURE_EXIT_CODE = 12;
     // FUTURE: This should come from configuration, NOT be hardcoded.
-    public static final int MAX_SIGNED_TXN_SIZE = 6144;
+    public static final int MAX_SIGNED_TXN_SIZE = 6145;
```

Comment:
This is interesting.  Any idea what added a byte?

Reply:
Oh no typo. I was testing and did that. Thanks for catching @jsync-swirlds

Reply:
I'm wondering why the byte count is one higher than expected.  Not a problem, trying to understand what we changed.